From bfbf7371d9bd3b6a668739cf0150c8c037a7abd4 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 18 Jun 2014 15:32:42 +0000
Subject: [PATCH 155/462] MANTIS 0019643 extension:
 Platform_Extension_CUSS_1_2: Single-application mode device support added
 property "JarJMockIt" for unit tests

git-svn-id: svn://localhost/SelfServiceBuild/trunk@228 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 7bda764..1c875ac 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -186,6 +186,9 @@
 <!-- junit -->
 <property name="JarJunit"                   value="${DIR_GLOBALS}/ThirdParty/junit/4.10/junit-4.10.jar"/>
 
+<!-- jMockIt -->
+<property name="JarJMockIt"                   value="${DIR_GLOBALS}/ThirdParty/jmockit/1.8/jmockit.jar"/>
+
 <!-- scxml -->
 <property name="JarSCXML"                   value="${DIR_GLOBALS}/ThirdParty/commons-scxml/0.9/commons-scxml-0.9.jar"/>
 <property name="JarLogging"                 value="${DIR_GLOBALS}/ThirdParty/commons-logging/1.1.1/commons-logging-1.1.1.jar"/>
-- 
2.41.0.windows.1

